tox.ini

```diff
@@ -15,6 +15,7 @@ python =
 [testenv]
 deps =
   mypy: mypy~=0.711
+  test: pytest-cov
```

Comment:
Please make the coverage a different environment: I want to run unit tests without coverage (assuming that coverage significantly slows down the unit tests, otherwise I don't really care). Maybe https://tox.readthedocs.io/en/latest/config.html#complex-factor-conditions can help, or maybe we can call the environment "TESTENVNAME-cov" and set the `--cov` option in a setenv to minimize code duplication in `tox.ini`.

Reply:
The setup is quite complex already.
I think we shouldn't increase the complexity until we really see some performance issues.

Reply:
Hello @Oberon00,
Eventually, I've switched to the approach that you suggested in order to use the editable mode.
Could you please re-review this PR?
Thank you!

Reply:
Why is editable mode required? Is that to more quickly iterate on unit tests?

I think it would be great to also benchmark the performance of the unit tests with coverage. In my experience it's not a huge overhead.

Reply:
> Why is editable mode required?

Answered in https://github.com/open-telemetry/opentelemetry-python/pull/128#discussion_r333939817.

> Is that to more quickly iterate on unit tests?

I didn't check, but it's not a reason anyway.